src/main/java/app/linguistai/bmvp/request/wordbank/QPredefinedWordList.java

```diff
@@ -0,0 +1,30 @@
+package app.linguistai.bmvp.request.wordbank;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
```

Comment:
Removing unused imports can improve the readability and mainatainability of the code